dace/frontend/python/newast.py

```diff
@@ -4426,7 +4430,7 @@ def visit_Call(self, node: ast.Call, create_callbacks=False):
                 arg = self.scope_vars[modname]
             else:
                 # Fallback to (name, object)
-                arg = (modname, self.defined[modname])
+                arg = modname
```

Comment:
Is this change handled already in the code somehow, i.e., is `self.defined` queried on demand?

Reply:
It’s a code path that we didn’t cover before in coverage. The rest of the code assumes a string return type rather than a tuple, and crashes somewhere else.